### ISSUE TITLE
Automatically detect dmypy in virtualenv/conda

### ISSUE THREAD

**Issue by omry** [dropbox/mypy-PyCharm-plugin#52]
Using virtualenv or conda, the user needs to manually select dmypy which is a bit annoying.
it would be great if the plugin could detect it automatically.

**Comment by jkleckner** [comment]
Yes, it is quite tedious to set up multiple virtual environments to use with a package and then to have to manually edit the path for each one and each time you change the environment for dmypy to be found there.  It seems that the virtual env path should be either prepended or appended to the path.  And I would think prepended.
